src/Apps/Conversation/Components/Conversations.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Sans, Serif } from "@artsy/palette"
+import { Conversations_me } from "__generated__/Conversations_me.graphql"
+import React from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { ConversationSnippetFragmentContainer as ConversationSnippet } from "./ConversationSnippet"
+
+interface ConversationsProps {
+  me: Conversations_me
+  relay: RelayRefetchProp
+}
+
+const Conversations: React.FC<ConversationsProps> = props => {
+  const { me } = props
+  const conversations = me.conversationsConnection.edges
+  return (
+    <Box px={1}>
+      <Serif size="5">Conversations</Serif>
+      {conversations.length ? (
+        conversations.map(edge => (
+          <ConversationSnippet conversation={edge.node} key={edge.cursor} />
+        ))
+      ) : (
+        <Sans size="2">No Conversations</Sans>
+      )}
+    </Box>
+  )
+}
+
+export const ConversationsFragmentContainer = createRefetchContainer(
```

Comment:
Interesting, this is a refetch container, but I don't see where it can get refreshed?

Reply:
i used it mainly for pagination but then ignored it for now.